geometry_msgs/Point position
---
float64 joint1_ang
float64 joint2_ang
float64 joint3_ang